### ISSUE TITLE
[Modular CLI] modular: error: invalid manifest: expiration has passed

### ISSUE THREAD

**Issue by benz0li** [modularml/mojo#3384]
### Issue description

`modular install mojo` results in `modular: error: invalid manifest: expiration has passed`.

Executing `modular install mojo` a second time results in `modular: error: invalid manifest: root json is missing or invalid.`.

### Steps to reproduce

On a host with docker installed:

```bash
docker run --rm -ti debian:12
```

In the container:

```bash
apt update \
  && apt -y install --no-install-recommends \
    apt-transport-https \
    ca-certificates \
    curl \
    gnupg
```

```bash
curl -s https://get.modular.com | sh -
```

```bash
modular install mojo
```

```bash
modular install mojo
```

### Version Info

```shell
- modular 0.9.1 (3460d12e)
- Debian 12 (bookworm)
```

**Comment by MariusDrulea** [comment]
I get exactly the same behavior with Ubuntu 22.04 LTS in WSL2.

**Comment by benz0li** [comment]
This is also reproducible with this repository's [`examples/docker/Dockerfile.mojosdk`](https://github.com/modularml/mojo/blob/8bd1dbdf26c70c634768bfd4c014537f6fdb0fb2/examples/docker/Dockerfile.mojosdk).

**Comment by DGSEM** [comment]
Same here with Ubuntu 22.04 LTS.

**Comment by slothyrulez** [comment]
Same Ubuntu 24

**Comment by ergonyc** [comment]
Broken for me as well on
- mac M2 _and_ 
- x86  Ubuntu 22.04.4 LTS

**Comment by nandor** [comment]
Broken on Debian Bookworm as well.

**Comment by ematejska** [comment]
Thank you for reporting, we're working on this.

**Comment by zbowling** [comment]
We should have a fix shortly. We had an internal key expire.

**Comment by smuniraj2018** [comment]
This has been resolved as of now with modular-cli 0.9.2 release. Please let us know if you still experience any issues,
